api/v4/source/system.yaml

```diff
@@ -339,19 +339,21 @@
           description: |
             Remove masked values from the exported configuration.
 
-            __Minimum server version__: 10.2.0
+            __Minimum server version__: 10.3.0
           required: false
           schema:
             type: boolean
+            default: false
```

Comment:
That should be default true no?

Reply:
@esarafianou It should be `false` to avoid breaking change. On the CLI, it was a new command so that's why we could set it to `true` by default. We are using an existing API here.

Reply:
Ah I see, thanks.